testshared/build.gradle

```diff
@@ -51,4 +51,5 @@ dependencies {
     implementation(Dependencies.androidx_fragment_testing) {
         exclude group: 'androidx.test', module: 'monitor' //fixes issue https://github.com/android/android-test/issues/731
     }
+    implementation Dependencies.kotlinx_coroutines_test
```

Comment:
Do we need this?

Reply:
I needed this in my `TestScheduler` but I got rid of it so no no we don't.